<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditView</class>
 <widget class="QWidget" name="CreditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Credit_Calc</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(36, 31, 49);
}</string>
  </property>
  <widget class="QRadioButton" name="radioButton_dif">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>80</y>
     <width>255</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Дифференцированный</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_an">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>55</y>
     <width>175</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Аннуитентный</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>20</y>
     <width>335</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Выберите график платежей:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>375</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Введите данные для рассчетов:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_percent">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>180</y>
     <width>370</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	background-color: rgb(198, 0, 218);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="prefix">
    <string>Процентная ставка: </string>
   </property>
   <property name="suffix">
    <string>%</string>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_month">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>235</y>
     <width>370</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
	background-color: rgb(198, 0, 218);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="suffix">
    <string> мес.</string>
   </property>
   <property name="prefix">
    <string>Срок: </string>
   </property>
   <property name="maximum">
    <number>1200</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_sum">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>290</y>
     <width>370</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	background-color: rgb(198, 0, 218);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="prefix">
    <string>Сумма кредита: </string>
   </property>
   <property name="suffix">
    <string> руб.</string>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>1000000000000000016976219238238959704141045173573106739630601035115997744067216908958262325956255112879408454231155599236459402033650892537856.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_result">
   <property name="geometry">
    <rect>
     <x>409</x>
     <y>45</y>
     <width>671</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit {
	background-color: rgb(152, 23, 156);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calc">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(174, 0, 255);
	color: rgb(255, 255, 255);
	border: 1px solid grey;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(168, 0, 241, 255));
}</string>
   </property>
   <property name="text">
    <string>Рассчет</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>140</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: rgb(255,255,255);
}</string>
   </property>
   <property name="text">
    <string>Результат:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_PaymentSchedule">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>180</y>
     <width>671</width>
     <height>192</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="rowCount">
    <number>5</number>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>130</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="pushButton_show_graph">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>400</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(174, 0, 255);
	color: rgb(255, 255, 255);
	border: 1px solid grey;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(168, 0, 241, 255));
}</string>
   </property>
   <property name="text">
    <string>График платежей</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>400</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(174, 0, 255);
	color: rgb(255, 255, 255);
	border: 1px solid grey;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(168, 0, 241, 255));
}</string>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
